view/concept-shared.twig

```diff
@@ -111,42 +111,7 @@
         </div></div></div>
         {% endif %}
       {% endfor %}
-      {% if concept.groupProperties %}
-      <div class="row">
-        <div class="property-label"><span class="versal property-click" title="{% trans "skosmos:memberOf_help" %}" >{{ 'skosmos:memberOf'|trans|upper }}</span></div>
-        <div class="property-value-column"><div class="property-value-wrapper">
-          <ul>
-          {% for grouppath in concept.groupProperties %}
-              <li>
-              {% for group in grouppath %}
-                <a class="versal" href="{{ group.uri | link_url(group.vocab,request.lang,'page',request.contentLang) }}">{% if group.notation %}<span class="versal">{{ group.notation }}</span>{% endif %}{{ group.label(request.contentLang) }}</a>{% if group.label.lang and (group.label.lang != request.contentLang or explicit_langcodes) %} <span class="versal">({{ group.label.lang }})</span>{% endif %}
-                {% if not loop.last %}<span class="versal"> &#62; </span>{% endif %}
-              {% endfor %}
-              </li>
-          {% endfor %}
-          </ul>
-        </div></div>
-      </div>
-      {% endif %}
-      {% if concept.arrayProperties %}
-      <div class="row">
-        <div class="property-label"><span class="versal property-click" title="{% trans "skosmos:memberOfArray_help" %}" >{{ 'skosmos:memberOfArray'|trans|upper }}</span></div>
-        <div class="property-value-column"><div class="property-value-wrapper">
-          <ul>
-          {% for grouppath in concept.arrayProperties %}
-            {% for group in grouppath %}
-              {% if group.type == 'skosmos:memberOfArray' %}
-                <li><a href="{{ group.uri | link_url(group.vocab,request.lang,'page',request.contentLang) }}">{% if group.notation %}<span class="versal">{{ group.notation }}</span>{% endif %}{{ group.label(request.contentLang) }}</a></li>
-              {% else %}
-              {% endif %}
-            {% endfor %}
-          {% endfor %}
-          </ul>
-        </div></div>
-      </div>
-      {% endif %}
-      {% set foreignLabels = concept.foreignLabels %}
-      {% if foreignLabels %}
+      {% if concept.foreignLabels %}
```

Comment:
Here one has made a bit too eager conflict resolve as twig variable `foreignLabels` is used later on in the template. I will fix this in a separate commit, pushed directly to master.

Reply:
Thanks @kouralex, good catch! I did the merge in GitHub UI as I thought it was trivial, but I didn't pay enough attention to this change.